components/PlantDetails/index.js

```diff
@@ -193,8 +245,15 @@ const StyledIconSection = styled.section`
   font-size: 14px;
   font-weight: normal;
 `;
+
+const StyledTagContainer = styled.ul`
+  display: flex;
+  justify-content:flex-start;
+`;
+
 const StyledEditDeleteSection = styled.section`
   display: flex;
   width: 100%;
   margin-top: 25px;
+  filter: grayscale(100%);
```

Comment:
❌
While this is a clever approach to make elements appear gray without touching the base color, from a accessibility point of view I highly recommend using a defined color value to have full control about the contrast.

Reply:
Thanks a lot for making us aware of this issue regarding a11y. We've fixed this using the Theme feature of styled components.